include/rocksdb/options.h

```diff
@@ -405,6 +405,13 @@ struct ColumnFamilyOptions {
   // Default: nullptr
   std::shared_ptr<const SliceTransform> prefix_extractor;
 
+  // If non-nullptr, memtable will use the specified function to extract
+  // prefixes for keys, and optimize CPU usages for sequential inserts within
+  // each prefix. Currently only affects the default skip-list-based memtable. 
```

Comment:
Should introduce in what use cases users can consider to use this feature and how should they determine the prefix.